packages/typespec-test/test/todo_non_branded/generated/typespec-ts/review/todo-non-branded.api.md

```diff
@@ -4,605 +4,239 @@
-export interface TodoItemsGet {
-    // (undocumented)
-    delete(options?: TodoItemsDeleteParameters): StreamableMethod<TodoItemsDelete200Response | TodoItemsDelete404Response>;
-    // (undocumented)
-    get(options?: TodoItemsGetParameters): StreamableMethod<TodoItemsGet200Response | TodoItemsGet404Response>;
+    createAttachment: (itemId: number, contents: TodoAttachment, options?: TodoItemsAttachmentsCreateAttachmentOptionalParams) => Promise<void>;
     // (undocumented)
-    patch(options: TodoItemsUpdateParameters): StreamableMethod<TodoItemsUpdate200Response>;
+    list: (itemId: number, options?: TodoItemsAttachmentsListOptionalParams) => Promise<PageTodoAttachment>;
 }
 
 // @public
-export interface TodoItemsGet200Response extends HttpResponse {
+export interface TodoItemsCreateOptionalParams extends OperationOptions {
     // (undocumented)
-    body: TodoItemOutput;
+    attachments?: TodoAttachment[];
     // (undocumented)
-    status: "200";
+    contentType?: string;
 }
 
 // @public
-export interface TodoItemsGet404Response extends HttpResponse {
-    // (undocumented)
-    status: "404";
-}
-
-// @public (undocumented)
-export type TodoItemsGetParameters = RequestParameters;
-
-// @public (undocumented)
-export interface TodoItemsList {
-    // (undocumented)
-    get(options: TodoItemsListParameters): StreamableMethod<TodoItemsList200Response>;
-    // (undocumented)
-    post(options: TodoItemsCreateJsonParameters): StreamableMethod<TodoItemsCreateJson200Response | TodoItemsCreateJson422Response>;
-    // (undocumented)
-    post(options: TodoItemsCreateFormParameters): StreamableMethod<TodoItemsCreateForm200Response | TodoItemsCreateForm422Response>;
+export interface TodoItemsDeleteOptionalParams extends OperationOptions {
 }
 
 // @public
-export interface TodoItemsList200Response extends HttpResponse {
-    // (undocumented)
-    body: TodoPageOutput;
-    // (undocumented)
-    status: "200";
-}
-
-// @public (undocumented)
-export type TodoItemsListParameters = TodoItemsListQueryParam & RequestParameters;
-
-// @public (undocumented)
-export interface TodoItemsListQueryParam {
-    // (undocumented)
-    queryParameters: TodoItemsListQueryParamProperties;
+export interface TodoItemsGetOptionalParams extends OperationOptions {
 }
 
-// @public (undocumented)
-export interface TodoItemsListQueryParamProperties {
-    limit: number;
-    offset: number;
+// @public
+export interface TodoItemsListOptionalParams extends OperationOptions {
+    limit?: number;
+    offset?: number;
 }
 
 // @public
-export interface TodoItemsUpdate200Response extends HttpResponse {
+export interface TodoItemsOperations {
     // (undocumented)
-    body: TodoItemOutput;
+    attachments: TodoItemsAttachmentsOperations;
     // (undocumented)
-    status: "200";
-}
-
-// @public (undocumented)
-export interface TodoItemsUpdateBodyParam {
+    create: (item: TodoItem, options?: TodoItemsCreateOptionalParams) => Promise<{
```

Comment:
This happens in quite a few places

Reply:
This is because of there is an @visibilty(create) and this caused we failed to get effective model there, if the visibility create is removed, this should be fixed.

Confirmed with Renhe as below
> No need to support visibility create for now. The spec will be updated.